rust/src/builders/tx_builder.rs

```diff
@@ -867,18 +640,14 @@ impl TransactionBuilder {
             certs: None,
             withdrawals: None,
             auxiliary_data: None,
-            input_types: MockWitnessSet {
-                vkeys: BTreeSet::new(),
-                scripts: BTreeSet::new(),
-                bootstraps: BTreeSet::new(),
-            },
             validity_start_interval: None,
             mint: None,
-            native_scripts: None,
             script_data_hash: None,
             collateral: None,
             required_signers: None,
```

Comment:
We have to redesign `required_signers` because we can't just call `set_required_signers` anymore. It has to take into account the fact that some required signers may already have been added from the input/mint/cert/withdrawal builders

Reply:
So I'll just remove it.

Reply:
BTW, how is `required_signers` used in transaction body? We've never set it in our app and there is no unit tests against it. My current implement is to convert remaining required witnesses into `required_signers`. Does it make sense?